.github/workflows/ci-ai.yml

```diff
@@ -1,16 +1,13 @@
 name: AI
 
 on:
+    pull_request:
     push:
         branches:
             - master
```

Comment:
Do we need to have it on push on master too?

Reply:
We do to have a baseline based on `master`. There should be fewer of these than WIP commits (though they _do_ run each 3x to average out)